internal/schema/schema_util.go

```diff
@@ -59,11 +59,11 @@ func formatDescription(name string, description string) string {
 	return strings.Join(resultLines, "\n")
 }
 
-// typeNameInTypes determines if a name is already present in a set of config.TypeConfig.
+// typeNameInTypes determines if a name is already present in a set of config.TypeConfig
 func typeNameInTypes(s string, types []config.TypeConfig) bool {
 	for _, t := range types {
-		if t.Name == s {
-			return true
+		if strings.EqualFold(t.Name, s) {
```

Comment:
This is arguably a future bug for us to fix (GraphQL is case-sensitive), however we munge the case so the types are exported in Go, and proper-case them (url => URL, etc).  Probably the _right_ thing to do in the future is store all types in the case they have in the schema, and convert to Go format in the templates with a pipeline or something.

Reply:
Ideally any case specific items for Go are handled in the Go generator, etc.

Reply:
they unfortunately aren't, and the work-around I had in config was to add type multiple times with multiple cases.  I agree this needs to be improved, but that is future work.